common/static/common/js/components/collections/paging_collection.js

```diff
@@ -25,8 +25,31 @@
                 // to the same object references for sortableFields and filterableFields.
                 this.sortableFields = {};
                 this.filterableFields = {};
+
+                this.paginator_core = {
+                    type: 'GET',
+                        dataType: 'json',
+                        url: function () { return this.url; }
+                };
+                var self = this;
```

Comment:
Nit: Put all variable definitions at the start of the function. This will be linted against shortly when we add JSCS to the platform.

http://jscs.info/rule/requireVarDeclFirst